src/main/webapp/i18n/en/exercise.json

```diff
@@ -187,7 +187,10 @@
                     "instructionDeleted": "Grading Instruction deleted:",
                     "scoringChange": "Scoring changed:",
                     "results": "The existing results must be recalculated!",
-                    "confirmtext": "Do you really want to perform these changes?"
+                    "confirmtext": "Do you really want to perform these changes?",
+                    "reevaluate": "Re-evaluate",
+                    "deleteFeedbacks": "Additionally delete all corresponding feedbacks that are used to assess submissions.",
```

Comment:
only feedback that is associated with the updated grading instruction is deleted right? Feeback is not used to assess submissions is it? Grading instructions are used to assess submission and feedback is given for the assessment.
I would say, this should be "Additionally delete all corresponding feedback that is associated with the deleted Grading Instruction"
I proposed a German translation for this above.

Reply:
yes exactly. This option does not delete all feedback, guess I missed that. one. The suggested sentence covers it properly. For the other question, so result have a list of feedback. When the total score is calculated, the score of feedback is taken into account.